dd-sdk-android/src/test/kotlin/com/datadog/android/utils/forge/ActionEventForgeryFactory.kt

```diff
@@ -68,6 +68,7 @@ internal class ActionEventForgeryFactory :
                 type = ActionEvent.ActionEventSessionType.USER,
                 hasReplay = forge.aNullable { aBool() }
             ),
+            source = ActionEvent.Source.ANDROID,
```

Comment:
Should we use `aValueFrom` here and in other event forgeries?

Reply:
Yes, here and in all the other events factories :) . I will change that, tks for spotting this